src/plugins/router.js

```diff
@@ -1,11 +1,17 @@
 import { health } from '../routes/health.js'
 import { apply } from '../routes/v1/apply/index.js'
+import { summaryLogsRoutes } from '../routes/v1/organisations'
 
 const router = {
   plugin: {
     name: 'router',
     register: (server, _options) => {
-      server.route([health, ...apply])
+      const summaryLogsRoutesBehindFeatureFlag =
+        process.env.FEATURE_FLAG_SUMMARY_LOGS === 'true'
```

Comment:
Is there a `config.get` based way of doing this (instead of going straight to `process.env`)

Reply:
ooo, yes you're right